sidebars.json

```diff
@@ -3,10 +3,7 @@
     {
       "label": "Quick Start",
       "collapsed": false,
-      "items": [
-        "getting-started",
-        "motivation"
-      ],
+      "items": ["getting-started", "motivation", "rooks"],
```

Comment:
I think rooks can be removed here. @qiweiii

Reply:
If we remove it, the sidebar will disappear when to go 'docs/' page.